api/nakadi-event-bus-api.yaml

```diff
@@ -1634,6 +1670,37 @@ definitions:
       purposes. We recommend logging this object in order to solve connection issues. Clients
       should not parse this structure.
 
+  CursorDistanceQuery:
+    required:
+      - partition
+      - initial_offset
+      - final_offset
```

Comment:
Nice idea. From programmer perspective this allows to directly use the stored cursors.